cmake/testing.cmake

```diff
@@ -196,20 +196,16 @@ if(VTK_VERSION VERSION_GREATER 9.0.1)
   f3d_test_interaction(TestInteractionAnimationMovement KameraAnim.glb "300,300" "--camera-index=1")#Space;MouseMovement;Space;
 
   f3d_test_long_timeout(TestHDRI8Bit suzanne.ply "300,300" "--hdri=${CMAKE_SOURCE_DIR}/data/testing/logo.tif")
-  set_tests_properties(TestHDRI8Bit PROPERTIES LABELS "HDRI")
-
   f3d_test_long_timeout(TestHDRIOrient suzanne.stl "300,300" "--up=+Z --hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
-  set_tests_properties(TestHDRIOrient PROPERTIES LABELS "HDRI")
-
   f3d_test_long_timeout(TestHDRIToneMapping suzanne.ply "300,300" "-t --hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
-  set_tests_properties(TestHDRIToneMapping PROPERTIES LABELS "HDRI")
 
   f3d_test_interaction(TestInteractionHDRIMove suzanne.ply "300,300" "--hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr") #Shift+MouseRight;
-  set_tests_properties(TestInteractionHDRIMove PROPERTIES LABELS "HDRI")
 
   # Test exit hotkey
   f3d_test_interaction_no_baseline(TestInteractionSimpleExit cow.vtp "300,300") #Escape;
-  set_tests_properties(TestInteractionSimpleExit PROPERTIES PASS_REGULAR_EXPRESSION "Interactor has been stopped, no rendering performed")
+  if(F3D_ENABLE_LONG_TIMEOUT_TESTS)
+    set_tests_properties(TestInteractionSimpleExit PROPERTIES PASS_REGULAR_EXPRESSION "Interactor has been stopped, no rendering performed")
+  endif()
```

Comment:
not sure to follow this

Reply:
if `F3D_ENABLE_LONG_TIMEOUT_TESTS` is disabled, `TestInteractionSimpleExit` test target does not exist and produce a CMake error